cmd/argocd/commands/context_test.go

```diff
@@ -35,6 +35,33 @@ users:
 - auth-token: vErrYS3c3tReFRe$hToken
   name: localhost:8080`
 
+const testCoreConfig = `contexts:
```

Comment:
Suggestion: Instead of creating a new constant, can a placeholder be added in the existing constant for the single value that is changing. The placeholder value can be inserted in individual test cases.

Reply:
Done